package.json

```diff
@@ -13,6 +13,7 @@
     "lint-fix": "vue-cli-service lint",
     "pretest": "rm -rf .nyc_output || true",
     "test": "npm run test-unit && npm run test-ci",
+    "check-peer-deps": "npm ls",
```

Comment:
It seems like `npm ls` generates a very long output; you could set the [`depth`](https://docs.npmjs.com/cli/ls.html#depth) to limit this, and it seems the behaviour would still be the same:
```console
npm ls --depth=0
```